src/scoring.jl

```diff
@@ -288,45 +288,36 @@ function ck_daily_workers_teams(
     worker_teams = map(x -> workers_info["team"][x], workers)
 
     teams_hourly = [
+        size(
         unique(
             team
             for (worker, team) in enumerate(worker_teams)
             if within(hour, shifts[day_shifts[worker]])
-        )
+        ), 1)
+        for hour = 1:24
+    ]
+    workers_hourly = [
+        [
+            worker 
+            for (num, worker) in enumerate(workers)
+            if within(hour, shifts[day_shifts[num]]) 
+        ]
         for hour = 1:24
```

Comment:
Uwzględniasz, że dzień jest od 7:7?

Reply:
Tak, ta tablica pod workers_hourly[___x___] trzyma listę pracowników, którzy o godzinie ___x___ są w pracy. Początek/koniec dnia mają znaczenie w pętli, gdzie decyduje to kolejności sprawdzania.